src/EluvioLive.js

```diff
@@ -1268,14 +1269,17 @@ Lookup NFT: https://wallet.contentfabric.io/lookup/`;
     return { signature, multiSig };
   }
 
-  async MakeAuthServiceRequest({ method, path, body }) {
-    const { signature, multiSig } = await this.Sign({
+  async PostServiceRequest({path, body }) {
```

Comment:
this is specifically for the 'authority' service (which is the only one with the 'signed body' auth)
didn't you want to have it say AuthorityServiceRequest?

Maybe PostAuthServiceRequest

Reply:
Will organize it later into it's own classes so didn't want to make too much of a distinction. I was also assuming all our Services would have the same consistent auth procedure.